### PR TITLE
Cache timestamps when using DruidBeams.fromConfig.

### PR DESCRIPTION
**Pull request by gianm**
~130% speedup in BenchmarkMain (150k/sec -> 350k/sec on my machine.)

Also add a benchmark class (in a separate commit).

### CODE REVIEW

**Comment by fjy** [comment]
:+1:

File: core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ import org.joda.time.chrono.ISOChronology
 import org.scala_tools.time.Implicits._
 import scala.util.Random
 
-class DruidBeamMaker[A: Timestamper](
+class DruidBeamMaker[A](
```
Comment (dclim) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala]: unused import com.metamx.tranquility.typeclass.Timestamper

Comment (gianm) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala]: removed.

**Comment by dclim** [comment]
:+1:

**Pull request merged**